src/main/java/core/basesyntax/serviceimpl/CalculateValues.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.strategyimpl.StrategyImpl;
+import java.util.List;
+
+public class CalculateValues {
```

Comment:
bad name, think better when naming something. Class name should be representation what this class does, CalculateValues is very general name